open_event/helpers/oauth.py

```diff
@@ -0,0 +1,22 @@
+class OAuth():
+    """Google Credentials"""
+
+    CLIENT_ID='449612261522-1eg34prt23l0454et59qgqno3rjd8muq.apps.googleusercontent.com'
+    CLIENT_SECRET='aq8XaUlxCfhwwMyZyNw8kS-D'
+    REDIRECT_URI='http://localhost:8001/gCallback'
+    AUTH_URI='https://accounts.google.com/o/oauth2/v2/auth'
+    TOKEN_URI='https://www.googleapis.com/oauth2/v4/token'
+    USER_INFO='https://www.googleapis.com/userinfo/v2/me'
+    SCOPE=['profile','email']
+
+class FbOAuth():
+	"Facebook Credentials"
+	
+	CLIENT_ID='1790977354468723'
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Found indentation with tabs instead of spaces](https://www.codacy.com/app/triplez/open-event-orga-server/file/2558623416/issues/source?bid=3233864&fileBranchId=3292627#l15)